CHANGELOG.rst

```diff
@@ -2,6 +2,21 @@
 Release Notes
 =============
 
+6.0.3
+=====
+
+Prelude
+_______
+
+- This release also includes bugfixes to the process agent. See .. _Diff: https://github.com/DataDog/datadog-process-agent/compare/5.23.1...6.0.3 
```

Comment:
github seems to have trouble parsing this as reST, can you try a format similar to this? https://github.com/DataDog/datadog-agent/blob/e242e12cf1f1babfdc9ee96c2d8572d4c027be89/CHANGELOG.rst#L168-L170